app/views/news/show.html.erb

```diff
@@ -27,24 +27,31 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 
-<% content_for :action_menu_specific do %>
-  <%= link_to(l(:button_edit),
-        edit_news_path(@news),
-        :class => 'icon icon-edit',
-        :accesskey => accesskey(:edit),
-        :onclick => 'Element.show("edit-news"); return false;') if User.current.allowed_to?(:manage_news, @project) %>
-  <%= watcher_link(@news, User.current) %>
-  <%= link_to(l(:button_delete),
-        news_path(@news),
-        :confirm => l(:text_are_you_sure),
-        :method => :delete,
-        :class => 'icon icon-delete') if User.current.allowed_to?(:manage_news, @project) %>
+<%= toolbar title: "#{avatar(@news.author)} #{@news.title}" do %>
```

Comment:
i'd like to add an `avatar` option to the `ToolbarHelper`, actuallly. While this could be done later, I'd like to avoid deleting existing functionality.